src/org/kie/jenkins/MavenSettingsConfig.groovy

```diff
@@ -0,0 +1,119 @@
+package org.kie.jenkins
```

Comment:
This is probably the case for builder pattern, this is pretty easy on groovy. Just create a new `MavenSettingsConfigBuilder` file like

```
import groovy.transform.builder.Builder
import groovy.transform.builder.ExternalStrategy

// New builder class for the Message class.
@Builder(builderStrategy = ExternalStrategy, forClass = MavenSettingsConfig)
class MavenSettingsConfigBuilder {}
```
and then you will be able to...
```
def mavenSettingsConfig = new MavenSettingsConfigBuilder()  // Create new MavenSettingsConfigBuilder instance.
        .settingsXmlConfigFileId('whatever')
        .settingsXmlPath('whatever')
...
        .build()  // Return new MavenSettingsConfig instance. 
```

there is even a simpler option here (no need to create a new class for it)

```
import groovy.transform.builder.Builder
 
@Builder
class MavenSettingsConfig {
```
and then
`new MavenSettingsConfig.Builder().settingsXmlConfigFileId('whatever').build()`

but up to you